packages/editor/src/components/provider/use-block-editor-settings.js

```diff
@@ -125,6 +125,9 @@ function useBlockEditorSettings( settings, hasTemplate ) {
 		);
 		const { canUser } = select( coreStore );
 
+		// Prefetch and parse patterns.
+		select( 'core/block-editor' ).__experimentalGetAllowedPatterns();
```

Comment:
This is very weird to me to be honest. Can we at least comment that it's a performance trick?

Reply:
I was trying to figure out a better location for this snippet. I'd be happy to move this somewhere else but I have no idea where to 🤔 I'll update the comment.